compliance_checker/ioos.py

```diff
@@ -565,8 +565,11 @@ def check_ioos_ingest(self, ds):
         m = ("To disallow harvest of this dataset to IOOS national products, "
              "global attribute \"ioos_ingest\" must be a string with value \"false\"")
         igst = getattr(ds, "ioos_ingest", None)
-        if igst is not None:
-            if igst not in ("true", "false"):
+        if igst is not None: # if not supplied, return default pass
```

Comment:
Please combine all these nested `if`statements using `and` statements -- short circuiting ensures that anything following a false condition won't fire and cause exceptions.  Also consider expanding your variable names `igst` -> `ingest`,  `r` -> `return_status`, etc.

Reply:
In fact you could just do `if isinstance(igst, str) and igst.lower() not in ("true", "false"):` since the not None implicitly is also done by the `isinstance` check.